.github/workflows/main_end_to_end_tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Staging smoke test
+
+# Only trigger, when the build workflow succeeded
```

Comment:
```suggestion
# Manual trigger, when the build workflow succeeded
```
This is manual only for main or did I miss something?

Reply:
You're 100% right. I copy/pasted, adjusted and forgot to clean up.